python/lsst/ts/salobj/domain.py

```diff
@@ -139,15 +140,19 @@ def __init__(self):
         # set of SalInfo
         self._salinfo_set = weakref.WeakSet()
 
-        host = os.environ.get("LSST_DDS_IP")
-        if host is None:
+        host_name = os.environ.get("LSST_DDS_IP")
+        if host_name is None:
             host = random.randint(1, MAX_RANDOM_HOST)
         else:
             try:
-                host = int(ipaddress.IPv4Address(host))
+                unsigned_host = int(ipaddress.IPv4Address(host_name))
             except ipaddress.AddressValueError as e:
-                raise ValueError(f"Could not parse $LSST_DDS_IP={host} "
+                raise ValueError(f"Could not parse $LSST_DDS_IP={host_name} "
```

Comment:
I guess the answer to my question above is here... I'm not entirely sure the best thing to do is to raise an exception. Seems like too harsh to prevent the communication from happening because you can't parse `LSST_DDS_IP`... But I can see it is useful to be able to uniquely identify the source.

Reply:
This error is raised if LSST_DDS_IP is set to a value that cannot be parsed as a dotted address. I think that's reasonable because it means the user intended to provide a useful value but it cannot be used. LSST_DDS_IP is not set to anything then the None case a bit higher up is used and a random value is chosen.